### PR TITLE
[zh-cn]Sync automated-tasks-with-cron-jobs.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Preview:
https://github.com/kubernetes/website/pull/35568/files?short_path=ad0fd7e#diff-ad0fd7ed79281af89137f8d453dc18def385d8d0a1371f262929dbd8ee11c1bb

### CODE REVIEW

File: content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 1]
```diff
@@ -52,15 +53,15 @@ CronJob 有一些限制和特点。
...
 
 <!-- steps -->
 
 <!--
-## Creating a Cron Job
+## Creating a CronJob
 
 Cron jobs require a config file.
-This example cron job config `.spec` file prints the current time and a hello message every minute:
+Here is a manifest for a CronJob that runs a simple demonstration task every minute:
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md:64]: The zh-cn text need to be changed.

File: content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 2]
```diff
@@ -204,54 +211,63 @@ You can read more about removing jobs in [garbage collection](/docs/concepts/arc
...
 The `.spec.schedule` is a required field of the `.spec`.
-It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format string, such as `0 * * * *` or `@hourly`, as schedule time of its jobs to be created and executed.
+It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format string, such as `0 * * * *` or `@hourly`,
+as schedule time of its jobs to be created and executed.
+
 -->
-### 时间安排 {#schedule}
+### 调度时间设置 {#schedule}
 
 `.spec.schedule` 是 `.spec` 中的必需字段。它接受 [Cron](https://en.wikipedia.org/wiki/Cron)
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md:255]:
```suggestion
`.spec.schedule` 是 `.spec` 中的必需字段。它接受 [Cron](https://zh.wikipedia.org/wiki/Cron)
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
This is a PR with a label of `language/zh`. It looks weird if the number of en changes is more than that of zh-cn.

File: content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md (hunk 2 above)
Comment (yanrongshi) [reply content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md]: You are really meticulous. Thank you for your review

File: content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 3]
```diff
@@ -52,20 +53,20 @@ CronJob 有一些限制和特点。
...
 
 Cron jobs require a config file.
-This example cron job config `.spec` file prints the current time and a hello message every minute:
+Here is a manifest for a CronJob that runs a simple demonstration task every minute:
 -->
 ## 创建 CronJob {#creating-a-cronjob}
 
 CronJob 需要一个配置文件。
-本例中 CronJob 的`.spec` 配置文件每分钟打印出当前时间和一个问好信息：
+这是一个 CronJob 的清单，它每分钟运行一个简单的演示任务：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md:69]:
```suggestion
以下是针对一个 CronJob 的清单，该 CronJob 每分钟运行一个简单的演示任务：
```
that 是 cronjob 的定语

**Review by windsonsea — CHANGES_REQUESTED** [review]
If you accepted a review comment, just click `Resolve conversation`. If not, give a feedback. So everyone will know you have changed fully or partially.

File: content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 4]
```diff
@@ -204,54 +211,63 @@ You can read more about removing jobs in [garbage collection](/docs/concepts/arc
...
+{{< note >}}
 <!--
-All modifications to a cron job, especially its `.spec`, are applied only to the following runs.
+If you modify a CronJob, the changes you make will apply to new jobs that start to run after your modification
+is complete. Jobs (and their Pods) that have already started continue to run without changes.
+That is, the CronJob does _not_ update existing jobs, even if those remain running.
 -->
-{{< note >}}
-对 CronJob 的所有改动，特别是它的 `.spec`，只会影响将来的运行实例。
+如果你修改了一个 CronJob，你所做的修改将只被应用到将来所运行的实例上，
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md:240]:
```suggestion
如果你修改了一个 CronJob，你所做的修改将只被应用到将来所运行的任务上，
```

File: content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 5]
```diff
@@ -204,54 +211,63 @@ You can read more about removing jobs in [garbage collection](/docs/concepts/arc
...
+对当前 CronJob 内处于运行中的 Job 集合（和 Job 里面的 Pod）不会产生任何变化，它们将继续运行。
+也就是说，对 CronJob 的修改不更新现有的实例，即使这些实例处于运行状态。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md:242]:
```suggestion
也就是说，对 CronJob 的修改不更新现有的任务，即使这些任务处于运行状态。
```

**Review by windsonsea — COMMENTED** [review]
Most `job` is 任务. Is any special to use `实例` here?

File: content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 6]
```diff
@@ -204,54 +211,63 @@ You can read more about removing jobs in [garbage collection](/docs/concepts/arc
...
 -->
 ## 编写 CronJob 声明信息 {#writing-a-cronjob-spec}
 
-像 Kubernetes 的其他配置一样，CronJob 需要 `apiVersion`、`kind` 和 `metadata` 字段。
-有关配置文件的一般信息，请参考
-[部署应用](/zh-cn/docs/tasks/run-application/run-stateless-application-deployment/) 和
-[使用 kubectl 管理资源](/zh-cn/docs/concepts/overview/working-with-objects/object-management/)。
+像 Kubernetes 的其他对象一样，CronJob 需要 `apiVersion`、`kind` 和 `metadata` 字段。
+有关 Kubernetes 对象及它们的{{< glossary_tooltip text="清单" term_id="manifest" >}}的更多信息，请参考
+[资源管理](/zh-cn/docs/concepts/cluster-administration/manage-deployment/)和
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md:228]: 注意换行，尽量不要在中文字符之间（包括链接文字）

```suggestion
有关 Kubernetes 对象及它们的{{< glossary_tooltip text="清单" term_id="manifest" >}}的更多信息，
请参考[资源管理](/zh-cn/docs/concepts/cluster-administration/manage-deployment/)和
```

File: content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 7]
```diff
@@ -204,54 +211,63 @@ You can read more about removing jobs in [garbage collection](/docs/concepts/arc
...
 ### Schedule
 
 The `.spec.schedule` is a required field of the `.spec`.
-It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format string, such as `0 * * * *` or `@hourly`, as schedule time of its jobs to be created and executed.
+It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format string, such as `0 * * * *` or `@hourly`,
+as schedule time of its jobs to be created and executed.
+
 -->
-### 时间安排 {#schedule}
+### 调度时间设置 {#schedule}
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md:253]: 通常在这个上下文中我们将“schedule”理解为排期表。

File: content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md [hunk 8]
```diff
@@ -264,6 +280,10 @@ The format also includes extended "Vixie cron" step values. As explained in the
 A question mark (`?`) in the schedule has the same meaning as an asterisk `*`, that is, it stands for any of available value for a given field.
 -->
 {{< note >}}
+<!--
+A question mark (`?`) in the schedule has the same meaning as an asterisk `*`, that is,
+it stands for any of available value for a given field.
+-->
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/job/automated-tasks-with-cron-jobs.md:286]: 279-281 should be deleted then?

**Pull request merged**